modules/virt-adding-templates-to-custom-namespace.adoc

```diff
@@ -0,0 +1,65 @@
+// Module included in the following assemblies:
+//
+// * virt/vm_templates/virt-deploying-vm-template-to-custom-namespace.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="virt-adding-templates-to-custom-namespace_{context}"]
+= Adding templates to a custom namespace
+
+[role="_abstract"]
+The Scheduling, Scale, and Performance (SSP) Operator deploys virtual machine templates to the `openshift` namespace by default. To also publish these templates in a custom namespace, set the `commonTemplatesNamespace` field in the `HyperConverged` custom resource (CR). After the templates sync to the custom namespace, you can modify or delete them there.
+
+[NOTE]
+====
+Do not edit templates in the `openshift` namespace. The SSP Operator reconciles that namespace and overwrites changes.
+====
+
+.Prerequisites
+* Install the {product-title} CLI `oc`.
+* Log in as a user with cluster-admin privileges.
+* You created a custom namespace for templates.
+
+.Procedure
+
+. Optional: Create the custom namespace if it does not already exist.
++
+[source,terminal]
+----
+$ oc create namespace <custom_namespace>
+----
+
+. Optional: View the list of templates in the `openshift` namespace.
++
+[source,terminal]
+----
+$ oc get templates -n openshift
+----
+
+. Edit the `HyperConverged` CR.
```

Comment:
```suggestion
. Open the `HyperConverged` CR in your default editor by running the following command:
```

just a suggestion for clarity; I think most of our procedures phrase it this way